recipe/conda_forge_ci_setup/build_utils.py

```diff
@@ -130,6 +130,8 @@ def upload_package(feedstock_root, recipe_root, config_file, validate, feedstock
         channels = _global_config["channels"]["targets"]
         source_channels = ",".join(_global_config["channels"]["sources"])
 
+    private_upload = specific_config.get("private_upload", False)
```

Comment:
Instead of pulling this out of the ci support config here, add a new flag to the command line tool and have smithy pass that flag when it makes the call to the command line util.

Reply:
Done!